src/panels/GeoLocationPanel/Search/GeoSearch.tsx

```diff
@@ -0,0 +1,107 @@
+import { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Text } from '@mantine/core';
+
+import { FilterList } from '@/components/FilterList/FilterList';
+import { generateMatcherFunctionByKeys } from '@/components/FilterList/util';
+import { useAppDispatch } from '@/redux/hooks';
+import { handleNotificationLogging } from '@/redux/logging/loggingMiddleware';
+import { LogLevel } from '@/types/enums';
+import { useAnchorNode } from '@/util/propertyTreeHooks';
+
+import { MatchedLocation } from './types';
+import {
+  calculateAltitudeEarth,
+  calculateAltitudeExtraTerrestial,
+  ET_URL,
+  fetchPlacesEarth
+} from './util';
+
+export function GeoSearch({
+  onClick,
+  onHover,
+  search
+}: {
+  onClick: (lat: number, long: number, altitude: number, name: string) => void;
+  onHover: (lat: number, long: number) => void;
+  search: string;
+  mah?: number;
+}) {
```

Comment:
```suggestion
interface Props {
  onClick: (lat: number, long: number, altitude: number, name: string) => void;
  onHover: (lat: number, long: number) => void;
  search: string;
}

export function GeoSearch({ onClick, onHover, search }: Props) {
```
Note that I removed `mah` from the props since it was seemingly not used